ambry-api/src/main/java/com/github/ambry/config/CloudConfig.java

```diff
@@ -271,6 +275,13 @@ public class CloudConfig {
   @Default("10")
   public final int cloudBlobCompactionShutdownTimeoutSecs;
 
+  /*
+   * Grace period for erasing blobs
+   */
+  public static final String CLOUD_COMPACTION_GRACE_PERIOD_DAYS = "cloudCompactionGracePeriodDays";
```

Comment:
Please be consistent with the naming, one is with ., one is CamelCase 
"cloud.compaction.dry.run.enabled";
"cloudCompactionGracePeriodDays";

Reply:
done, thank you